kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/schemaregistry/SchemaRegistryAwareRecordSerDe.java

```diff
@@ -260,7 +260,7 @@ private String convertSchema(SchemaMetadata schema) {
   private Optional<Integer> extractSchemaIdFromMsg(ConsumerRecord<Bytes, Bytes> msg, boolean isKey) {
     Bytes bytes = isKey ? msg.key() : msg.value();
     ByteBuffer buffer = ByteBuffer.wrap(bytes.get());
-    if (buffer.get() == 0 && buffer.remaining() > 4) {
+    if (buffer.remaining() > 5 && buffer.get() == 0) {
```

Comment:
not connected with issues, just small fix

Reply:
can we extract these magic numbers as a constants please?